src/phases/base/battle-phase.ts

```diff
@@ -1,12 +1,13 @@
 import { globalScene } from "#app/global-scene";
 import { Phase } from "#app/phase";
 import { TrainerSlot } from "#enums/trainer-slot";
+import { playTween } from "#utils/anim-utils";
 
 /**
  * Adds functions to display and hide the enemy trainer
  */
 export abstract class BattlePhase extends Phase {
-  public showEnemyTrainer(trainerSlot: TrainerSlot = TrainerSlot.NONE): void {
+  public async showEnemyTrainer(trainerSlot: TrainerSlot = TrainerSlot.NONE): Promise<void> {
```

Comment:
```suggestion
  /** Slides the enemy trainer into view */
  public async showEnemyTrainer(trainerSlot: TrainerSlot = TrainerSlot.NONE): Promise<void> {
```
(if that's indeed what it does)